Cargo.toml

```diff
@@ -25,6 +25,8 @@ url = "*"
 traitobject = "*"
 typeable = "*"
 
+[dependencies.solicit]
+git = "https://github.com/mlalic/solicit"
```

Comment:
Yeah, not yet... I wanted to maintain some flexibility, without pushing multiple versions out to crates.io, if it turned out that I needed to do some changes there to make things work better in `hyper` (so far it turned out that I didn't).

Of course it's something to do before finally landing this to make sure `hyper` is pinned to a proper version.